_includes/scenarios/integrate/establish-a-relationship-to-another-identity.md

```diff
@@ -0,0 +1,164 @@
+
+Keeping the `id` of the Relationship and the `id` of the associated RelationshipChange in mind is useful if you want to return to the created Relationship later in order to retrace changes to the Relationship.
+{: .notice--info}
+
+### Send it as an App user
+
+As already mentioned in the description of the [input for creating a RelationshipTemplate]({% link _docs_integrate/establish-a-relationship-to-another-identity.md %}#input-for-creating-a-relationshiptemplate), a RelationshipTemplateContent must be used as the value of the `content` property of the [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) if you intend to use the RelationshipTemplate to establish a Relationship between the Templator and an App user. Assuming that there is no Relationship between them yet, the App user receives the Request specified in the `onNewRelationship` property of the [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent) after scanning the QR Code associated with the RelationshipTemplate. The App user has the option of accepting or rejecting the Request. If they accept the Request appropriately, a Relationship Request is sent to the Templator. Internally, this corresponds to the creation of a data object of type [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) with an associated RelationshipChange and `"Pending"` as `status`. The [RelationshipChange]({% link _docs_integrate/data-model-overview.md %}#relationshipchange) has `"Creation"` as `type` and `"Pending"` as `status`. The `request.content.response` property of the RelationshipChange contains the Response of the App user to the Request.
+
+Please note that the general procedure is the same if an App user instead of a Connector wants to send a Relationship Request to the Templator. The difference is that the [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) does not need to contain a RelationshipTemplateContent in its `content` property if it is intended to be used by a Connector.
+{: .notice--info}
+
+## Establish an active Relationship
+
+After the Requestor has sent the Relationship Request, the Integrator of the Templator can accept it if they want to establish an active Relationship to the Requestor. We now explain all required steps for establishing an active Relationship, including the necessary synchronization of the Templator and any other Connector that may be involved at certain points in time. Please note that the synchronization can also be automated by using the [Sync Module]({% link _docs_operate/modules.md %}#sync).
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/de35cc09-894e-431f-819a-33bc7363ea30" id="6y2O3PCGpyGD"></iframe></div>
```

Comment:
Can you add in the diagram, that the Requestor is informed about the acceptance of the Relationship Request, such that their status of the Relationship is also updated?

Reply:
Difficult to add this in the diagram for both cases (Requestor could be Connector or App user) at the same time, but I've added the section "Be informed about the acceptance of the Relationship Request" and a corresponding diagram now.